package.json

```diff
@@ -27,7 +27,7 @@
     "dev-nowatch": "bash -c 'NODE_ENV=development node server/server.js & NODE_ENV=development webpack-dev-server'",
     "test-accessibility": "./test/accessibility.sh",
     "test-unit": "yarn test-unit:components && yarn test-unit:store && yarn test-unit:app",
-    "test-unit:app": "mocha --require mock-local-storage --file test/unit/helpers/babel-register.js --file test/unit/helpers/init.js \"test/unit/**/*.test.js\"",
+    "test-unit:app": "mocha  --exit --require mock-local-storage --file test/unit/helpers/babel-register.js --file test/unit/helpers/init.js \"test/unit/**/*.test.js\"",
```

Comment:
It's unclear why the tests didn't exit anymore without `--exit` flag. It's probably because there are some open handles somewhere but I couldn't debug the problem and not sure if it is serious or not.